docarray/typing/url/image_url.py

```diff
@@ -1,17 +1,18 @@
 import io
 import struct
-from typing import TYPE_CHECKING, Any, Optional, Tuple, Type, TypeVar, Union
+from typing import TYPE_CHECKING, Optional, Tuple, TypeVar, Union, Type, Any
 
 import numpy as np
 
+from pydantic import BaseConfig
+from pydantic.fields import ModelField
+
```

Comment:
lets move this to TypeChecking (where it was before) as we don't need them at runtime but only when doing type checking